usr/sbin/truenas-install

```diff
@@ -970,24 +970,6 @@ menu_install()
     echo "deb http://apt.tn.ixsystems.com/truenas/unstable/ truenas main" \
 	    > /tmp/data/etc/apt/sources.list
 
-    # Always import zfs pool
-    cat >/tmp/data/etc/systemd/system/zfs-import-bootpool.service << EOF
-[Unit]
-DefaultDependencies=no
-Before=zfs-import-scan.service
-Before=zfs-import-cache.service
-
-[Service]
-Type=oneshot
-RemainAfterExit=yes
-ExecStart=/sbin/zpool import -N -o cachefile=none freenas-boot || true
```

Comment:
This is not a shell script, '||' and 'true' were passed as arguments to `zpool` resulting in bad arguments error.